frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -57,6 +57,7 @@ import { StyledResizableContainer } from "./styled-components"
 import Tooltip from "./Tooltip"
 
 import "@glideapps/glide-data-grid/dist/index.css"
+import "@glideapps/glide-data-grid-cells/dist/index.css"
```

Comment:
What's this import for?

Reply:
The styling is transformed into this css on glide-data-grid side. To apply the configured styles for cells from the cells package, we need to import the CSS.